src/applications/facility-locator/components/SearchControls.jsx

```diff
@@ -55,22 +70,46 @@ class SearchControls extends Component {
     this.props.onSubmit();
   };
 
+  hideClearInput = () => {
+    this.inputClearRef.current.style.display = 'none';
+    this.inputRef.current.value = '';
+    focusElement('#street-city-state-zip');
+  };
+
+  renderClearInput = () => {
+    if (window.Cypress || !environment.isProduction()) {
+      return (
+        <i
+          aria-hidden="true"
+          className="fas fa-times-circle fa-clear"
+          id="clear-input"
+          ref={this.inputClearRef}
+          onClick={this.hideClearInput}
+        />
+      );
+    }
+    return null;
+  };
   renderLocationInputField = currentQuery => (
     <>
       <label htmlFor="street-city-state-zip" id="street-city-state-zip-label">
         City, state or postal code{' '}
         <span className="vads-u-color--secondary-dark">(*Required)</span>
       </label>
-      <input
-        id="street-city-state-zip"
-        name="street-city-state-zip"
-        style={{ fontWeight: 'bold' }}
-        type="text"
-        onChange={this.handleQueryChange}
-        value={currentQuery.searchString}
-        title="Your location: Street, City, State or Postal code"
-        required
-      />
+      <div className="input-clear">
+        {this.renderClearInput()}
```

Comment:
If we do this, I don't think we need `handleInputClear`: 

```suggestion
        {currentQuery.searchString.length > 0 && this.renderClearInput()}
```